.mailmap

```diff
@@ -47,4 +47,7 @@ Tom Gale <tom.gale@bom.gov.au> <tom@tjtg.net>
 Victoria Smart <victoria.smart@metoffice.gov.uk> <25484420+VictoriaLouiseS@users.noreply.github.com>
 Tim Pillinger <tim.pillinger@metoffice.gov.uk> <tim.pillinger@metoffice.gov.uk>
 Ying Zhao <91048290+yzhaobom@users.noreply.github.com> <91048290+yzhaobom@users.noreply.github.com>
-Zhiliang Fan <71359633+zfan001@users.noreply.github.com> <71359633+zfan001@users.noreply.github.com>
\ No newline at end of file
+Zhiliang Fan <71359633+zfan001@users.noreply.github.com> <71359633+zfan001@users.noreply.github.com>
+Anzer Khan <anzer.khan@metoffice.gov.uk> <anzer.khan@metoffice.gov.uk>
```

Comment:
This file is alphabetically sorted on first name - so please move your name up the list.